python/pyarrow/tests/conftest.py

```diff
@@ -184,9 +184,16 @@ def gcs_server():
     args = [sys.executable, '-m', 'testbench', '--port', str(port)]
     proc = None
     try:
+        # check first if testbench module is available
+        import testbench  # noqa:F401
+        # start server
         proc = subprocess.Popen(args, env=env)
-    except OSError as e:
+        # Make sure the server is alive.
+        assert proc.poll() is None
```

Comment:
Moving the assert is not strictly speaking necessary anymore, but still cleaner, because with the current setup this can just fail during the setup of the `gcsfs` fixture, which unnecessarily errors the respective tests, rather than just skipping them.